content/post/36-a-look-at-windows.md

```diff
@@ -0,0 +1,52 @@
+
+### Access errors / Garbage files {TODO: more appropriate word than "garbage"}
+![garbage](img/garbage.png)  
+
+Previously, there was a particularly bad bug that often caused failure for many flatfs (`go-ipfs`'s default storage method) operations, sometimes garbage files would be produced in the working directory as well.  
+The details of which can be found in the [Windows initiative issue](https://github.com/ipfs/go-ipfs/issues/4808), the short version is that we were trying to move data blocks from a temporary location to a target location, but the target location was being mangled into garbage. {TODO: still sounds too technical; see if we can avoid the word "mangled" as well}
+
+Now, flatfs operations should succeed without the chance of creating garbage files.
+
+### File output names
+{TODO:[MD] numbered list + headers don't render correctly on ipfs.io}  
+Previously, there was no filtering applied when extracting files out of IPFS. This caused multiple issues.  
+1. #### Extracting content from IPFS could fail due to native file system restrictions
+![filenames](img/filenames-crop.png)  
+FAT32, NTFS, ReFS, and Windows itself, all impose their own path limitations. If a hash contained a non-legal path, operations like `ipfs get` would fail. In the image above, the [XKCD archive ](https://github.com/ipfs/archives/issues/21) could not be downloaded on Windows because the directory "1031 - s keyboard leopard " ends in a space.  
+
+2. #### Hashes that contained platform-specific malicious files and symlinks could be extracted
+![overwrite 1](img/overwrite%201.png)  
+![overwrite 2](img/overwrite%202.png)  
+{TODO: This is a minor(currently impractical and easily detected) security concern but a security concern nonetheless, consider what tone to use here}
+
+Previously, it was possible for users to craft specific hashes that could escape the extraction root and overwrite files, if the target file's location was known in advanced. In addition, symlinks who's target escaped the hash-root (either via relative or absolute means) were allowed to be created.
+
+Now, we account for these foreign limitations and translate pathnames into platform legal paths that are restricted to their root.
+
+### `stdio`
+![pipe](img/pipe.png)  
+Previously, stdin support was explicitly disabled in `go-ipfs`, preventing basic [IPC](https://en.wikipedia.org/wiki/Inter-process_communication) with other applications.
+Now, stdin support has been added to the Windows version of go-ipfs which allows you to place ipfs anywhere in a [pipeline](https://en.wikipedia.org/wiki/Pipeline_(Unix)).  
+
+***
+In addition to all of the above, there have been many other contributions to `go-ipfs` as well as to the tools it utilizes, both Windows specific and generic. As we move forward, the plan is to continue our efforts of improving and providing a *consistent* experience for `go-ipfs` users.
```

Comment:
Would it be correct and useful to link to the Windows initiative issue and say something like:

> There’s still a lot more to improve for IPFS on Windows. If you’re interested in contributing (or just keeping an eye on progress), check [the Windows initiative issue](#linktoissue) on `go-ipfs`.